src/components/availability-label/helper.ts

```diff
@@ -37,24 +93,13 @@ export const useAvailabilityData = ({
     }
   });
 
-  useGetV1ProductsIdentifier(isbn ?? "", {
-    query: {
-      // Publizon / useGetV1ProductsIdentifier is responsible for online
-      // materials. It requires an ISBN to do lookups.
-      enabled: isOnline && isbn !== null,
-      onSuccess: () => {
-        // For now we always consider online materials available at publizon
-        // as available. In the future this can be improved by checking
-        // 1. Whether the material has a quota (res?.product?.costFree)
-        // 2. If the product has a quota:
-        //    1. If the user has any quota loans available for the material type
-        //    2. If the library has a queue on the material
-        setIsAvailable(true);
-      }
-    }
-  });
+  useEffect(() => {
+    setIsLoading(
+      !!(isLoadingAvailability || isLoadingIdentifier || isLoadingProductInfo)
+    );
+  }, [isLoadingAvailability, isLoadingIdentifier, isLoadingProductInfo]);
```

Comment:
I like this construction but is it necessary to cast it to a boolean? Isn't the loading status always boolean from the different services?